docs/index.md

```diff
@@ -4,6 +4,20 @@ layout: default
 sort: 1
 ---
 
-# Introduction
+# Node Feature Discovery operator
 
-***COMING SOON.***
+The Node Feature Discovery operator manages detection
+of hardware features and configuration in a kubernetes cluster.
+The operator orchestrates all resources needed to run the
+[Node-Feature-Discovery](https://github.com/kubernetes-sigs/node-feature-discovery)
+DaemonSet
```

Comment:
Maybe drop the Daemonset here? Just *The operator orchestrates all resources needed to run [Node-Feature-Discovery](https://github.com/kubernetes-sigs/node-feature-discovery)* ?